charter/index.html

```diff
@@ -288,7 +288,7 @@ <h3>
             If additional in-scope Recommendation-track deliverables need to be added to the Charter before the Charter expires, the Working Group will prepare an updated Charter that differs only in deliverables.
           </p>
           <p>
-            The Working Group will not adopt new proposals until they have matured through the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> or another similar incubation phase.
+           The Working Group will adopt proposals that fulfill the requirements of <a href="https://www.w3.org/Guide/standards-track/#criteria">W3C Recommendation Track Readiness</a>. In particular, the Working Group will only adopt proposals it deems sufficiently mature. Proposals can be made to the Working Group through the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> or another incubation phase.
```

Comment:
```suggestion
           The Working Group will adopt proposals that fulfill the requirements of <a href="https://www.w3.org/Guide/standards-track/#criteria">W3C Recommendation Track Readiness</a>. In particular, the Working Group will only adopt proposals it deems sufficiently mature. Proposals can be made to the Working Group through the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> or another similar incubation phase.
```

I think *similar* is essential here. Quoting from https://www.w3.org/Guide/standards-track/#criteria

> This consensus could emerge from an incubation phase in WICG or another CG, or in a WG that has an established culture of taking and vetting suggestions from its public mailing list. 

If *similar* is still to vague we could even quote: "*that has an established culture of taking and vetting suggestions from its public mailing list.*", understanding that public github repositories can be considered comparable to mailing list.

Reply:
Wholeheartedly agree. "Similar", "comparable" ... in other words: any incubation process achieving the aforementioned readiness criteria (addresses real problem, formulates measureable solution, based on consensus in social community, mature enough to expect implementaitons).

Reply:
I agree to keep "similar" in as that entails a particular meaning and expectation that's taken in the context of the Readiness Criteria. I don't have a strong opinion on unpacking that further or quoting the BP, but if that's introduced here, that'd be fine too - although it'd be unnecessarily verbose. Charters are almost never intended to be taken raw without the context / references in mind. So, for me, "similar" is sufficient, as it was for other groups as well.

@barath  , can you please incorporate the suggested change to add "similar". Happy to approve the PR based on that.

Reply:
I would also rather keep "similiar" in the sentence, as suggested by @elf-pavlik [above](https://github.com/solid/solid-wg-charter/pull/48#discussion_r1304521418)